### PR TITLE
feat: welcoming shaun to the dev-team

### PR DESCRIPTION
**Pull request by raisedadead**
🎉

### CODE REVIEW

**Review by naomi-lgbt — DISMISSED** [review]
With pleasure.

**Comment by Sembauke** [comment]
Congratulations @ShaunSHamilton 🥇

File: docs/FAQ.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ If you have queries about the stack, architecture of the codebase, translations,
 | Nicholas Carrigan     | [@nhcarrigan](https://forum.freecodecamp.org/u/nhcarrigan)                   |
 | Oliver Eyton-Williams | [@ojeytonwilliams](https://forum.freecodecamp.org/u/ojeytonwilliams)         |
 | Rafael D Hernandez    | [@RafaelHernandez](https://forum.freecodecamp.org/u/rafaelhernandez)         |
+| Shaun Hamilton        | [@sky020](https://forum.freecodecamp.org/u/sky020)         |
```
Comment (ShaunSHamilton) [inline docs/FAQ.md:84]:
```suggestion
| Shaun Hamilton        | [@sky020](https://forum.freecodecamp.org/u/sky020)                          |
```

**Review by ShaunSHamilton — COMMENTED** [review]
Seems wrong to comment this, but perhaps just my OCD kicking in 😆

**Review by naomi-lgbt — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Pull request merged**